test/common/cdp.py

```diff
@@ -221,7 +221,7 @@ def start(self):
             self._browser_home = tempfile.mkdtemp()
             environ = os.environ.copy()
             environ["HOME"] = self._browser_home
-            environ["LC_ALL"] = "C.utf8"
+            environ["LC_ALL"] = "en_US.UTF-8"
```

Comment:
I'm afraid this won't work in our current tasks containers just yet, as they don't have any locales defined:
```
❱❱❱ locale -a
C
C.utf8
POSIX
```
If that is the right workaround, then we need to install `glibc-langpack-en` into the tasks container.

Reply:
FTR: I tested that in my toolbox (which *is* the cockpit/tasks container) with `LC_ALL=en_US.UTF-8 HOME=/tmp/f firefox`. I get a lot of errors on the console like
```
(firefox:237002): Gtk-WARNING **: 17:13:07.810: Locale not supported by C library.
	Using the fallback 'C' locale.
```
but the string comparison returns -1 in the developer console.